### ISSUE TITLE
Clean up build

### ISSUE THREAD

**Issue by theazureshadow** [blendersfun/seattle-theatre-site#1]
Most dependencies should be able to be installed by `npm install`. Generally there's less reason to have non-top-level dependencies installed globally, as is recommended in the `README`. I think `jsx` probably makes sense as a global rather than a local dependency, but it's probably still not necessary.

**Comment by blendersfun** [comment]
Yeah, seems sensible. Global installs seemed like the easiest way given that I wasn't ready to tackle setting up the gulp build yet.
